samples/cpp/speech_sample/main.cpp

```diff
@@ -85,6 +85,12 @@ int main(int argc, char* argv[]) {
         // --------------------------- Step 1. Initialize OpenVINO Runtime core and read model
         // -------------------------------------
         ov::Core core;
+        try {
+            const auto& gnaLibraryVersion = core.get_property("GNA", ov::intel_gna::library_full_version);
+            slog::info << "Detected GNA Library: " << gnaLibraryVersion << slog::endl;
+        } catch (std::exception& e) {
+            slog::info << "Can not detect GNA Library version, exception: " << e.what() << slog::endl;
```

Comment:
why can we fail to detect GNA version?

Reply:
> why can we fail to detect GNA version?
It should not, but I wanted to make it more safe for potential (future?) bugs.
Do you think making it more safe is big problem and I should remove try block?

Reply:
I think one potential problem may be on integration side - the GNA Plugin may not load (eg., due to dll not found), but we could still use speech_sample with CPU plugin.

Reply:
@ilya-lavrenov can we resolve this issue?